cpu/esp32/vendor/esp-idf/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_ESP_IDF
+    bool
+    depends on TEST_KCONFIG
+    depends on HAS_ARCH_ESP32
+    default y
+    select MODULE_ESP_IDF_DRIVER
+    select MODULE_ESP_IDF_ESP32
+    select MODULE_ESP_IDF_SOC
+    help
+      Espressif IoT Development Framework.
+
+rsource "driver/Kconfig"
+rsource "esp32/Kconfig"
+rsource "ethernet/Kconfig"
+rsource "heap/Kconfig"
+rsource "nvs_flash/Kconfig"
+rsource "soc/Kconfig"
+rsource "spi_flash/Kconfig"
```

Comment:
Is `wpa_supplicant/Kconfig` missing by intention? I should be almost if not the same as for ESP8266.

Reply:
I didn't yet get to the networking part, as that is coming later for most of cpus and drivers, due to lack of modelling of related modules, (e.g. netdev).

Reply:
OK, I was just wondering because you did it for EPS8266.

Reply:
Do you mind to add at least the module definition for `wpa_supplicant` as for ESP8266 so that the vendor libraries are consistent and we can later rely on that when we model the network interfaces like `esp_now` and `esp_wifi`?

Reply:
I added all `wpa_supplicant` modules in c364ca7185b0663ba513ff2ac838e3d7c86502f6